atest/robot/output/xunit.robot

```diff
@@ -71,6 +84,43 @@ Skipping non-critical tests is deprecated
     Run tests    --xUnit xunit.xml --xUnitSkipNonCritical     ${PASS AND FAIL}
     Stderr Should Contain   Command line option --xunitskipnoncritical has been deprecated and has no effect.
 
+Nullish timestamp prerebotmodifier
+    Nullish Timestamp Should Be    ${EMPTY}
+    
+Invalidated suite status
+    Run Tests Without Processing Output    -l log.html    ${PASS AND FAIL}
+    Remove Suite Status Attribute    ${OUTFILE}    name=starttime
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    --xUnit xunit.xml -l log.html    ${OUTFILE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${EMPTY}
+
+Missing suite status
+    Run Tests Without Processing Output    -l log.html    ${PASS AND FAIL}
+    Remove Suite Status Element    ${OUTFILE}
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    --xUnit xunit.xml -l log.html    ${OUTFILE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${EMPTY}
```

Comment:
These two tests seem to mainly validate that Rebot can parse output.xml files with missing information. I don't think it has anything to do with xunit files and it would probably be best to just remove these tests.

Reply:
True. These are more like a play around and learn the system. Will remove.